src/index.js

```diff
@@ -1,17 +1,15 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import ReactDOM from 'react-dom';
```

Comment:
- [ ] Kindly ensure a thorough check up of your app to avoid wasting review counts. This import does not work. You should be importing from `react-dom/client`.

  ![image](https://github.com/david30-maker/hello-world-react-two/assets/36542986/4cc21121-3e88-46d4-b261-6b4647ed8c0c)